app/server/models/credit_transfer.py

```diff
@@ -112,6 +113,8 @@ def resolve_as_completed(self, existing_blockchain_txn=None, queue='high-priorit
         if self.fiat_ramp and self.transfer_type in [TransferTypeEnum.DEPOSIT, TransferTypeEnum.WITHDRAWAL]:
             self.fiat_ramp.resolve_as_completed()
         if not existing_blockchain_txn:
+            self.blockchain_task_uuid = str(uuid4())
+            db.session.commit()
```

Comment:
committing deep inside processes often causes weird side-effects (the org filter can get angry) - is it possible to push this up higher?

Reply:
@michielderoos just checking in on this one bit